### ISSUE TITLE
HPC (High Performance Computing) around SymbiFlow stack

### ISSUE THREAD

**Issue by manili** [SymbiFlow/ideas#57]
Hello everyone,
I'm wondering to know what do you think about a project like SDAccel, but this time around SymbiFlow stack?

Best regards,
Manili

**Comment by manili** [comment]
Here is something like what I'm thinking about:
![SDAccel in SymbiFlow](https://user-images.githubusercontent.com/7406234/93202596-30773680-f768-11ea-88d7-acfe52836d4c.png)

**Comment by mithro** [comment]
You might want to look at what Andre De'Hon and @andrewb1999 are doing around Rosetta;
 * http://ic.ese.upenn.edu/abstracts/prflow_fpt2019.html
 * https://andrewb1999gsoc.blogspot.com/2020/08/gsoc-final-code-submission.html

**Comment by andrewb1999** [comment]
@manili Yes, I think a flow like this has a lot of potential and it has been a focus of mine for a bit now.  Andre and I are mostly interested in mapping to separate pr regions within a single FPGA, but I'm sure it's not too difficult to instead partition to multiple FPGAs.

Currently we are still using Vivado HLS while trying to get lower level details in symbiflow working.  Do you have any experience with using Bambu HLS?  We are very interested in switching to an open source HLS tool soon to have a fully open source c to bitstream flow.

**Comment by mithro** [comment]
You might also want to explore the newly released XLS from Google (https://github.com/google/xls) too.
